doc/nrf/libraries/bluetooth_services/mesh/light_ctrl.rst

```diff
@@ -23,6 +23,7 @@ For types common to all models, see :ref:`bt_mesh_models`.
 
    light_ctrl_srv.rst
    light_ctrl_cli.rst
+   light_ctrl_regulator.rst
```

Comment:
When the regulator is extracted from the light ctrl srv model, it will still be a part of the library, right? But it is not a model, or a part of the model then? Or is it?
Just trying to figure out if the placement is optimal or if it rather could/should be moved to another place. If it is more of a standalone thing now, maybe it could be placed directly under [mesh profile](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/bluetooth_services/mesh.html) subpages.